### DISCUSSION TITLE
MQTT to trigger recording (PIR detection simulation)

### DISCUSSION THREAD

**Discussion by Raptor607** [QuantumEntangledAndy/neolink#270] [Q&A]
Is there any current/planned functionality to able to trigger a "recording" or a "simulated PIR detection" on a camera when neolink receives an MQTT command?  I'm aware of the control/wakeup functionality, but it only wakes up the camera.  The scenario I'm encountering is that the PIR is not getting triggered by the subject because it is not close enough. I'd like to use Blue Iris to send an MQTT to neolink to begin wake AND record when a spotter (non-neolink) camera is triggered.

**Comment by QuantumEntangledAndy** [comment]
Probably would have to trigger a manual record since I can't really touch the auto record stuff, which means finishing the SD card code. I don't really have time for any new features at the moment but if you want to tackle it please do
